platform/darwin/test/MGLStyleLayerTests.h

```diff
@@ -6,5 +6,15 @@
 
 @property (nonatomic) IBOutlet MGLMapView *mapView;
 @property (nonatomic) XCTestExpectation *expectation;
+@property (nonatomic, copy, readonly, class) NSString *layerType;
```

Comment:
I suppose we dropped support for Xcode 7? Great use for the new class property. 👍  Does this mean it's safe to use not only in tests?